### ISSUE TITLE
Add support for PHP 5.5

### ISSUE THREAD

**Issue by danbohea** [hashbangcode/vlad#128]
Feedback from DrupalCamp Brighton included interest in PHP 5.5 from several peeps; indeed to those people the lack of support was a blocker to using Vlad at all.

I think it's a great suggestion, we're inevitably heading in this direction anyway, it's more a question of when than whether.

**Comment by wizonesolutions** [comment]
+1 because Drupal 8

**Comment by danbohea** [comment]
@wizonesolutions See #130.

**Comment by wizonesolutions** [comment]
@danbohea I saw the dev branch but it didn't seem to have all of regular dev in it. I tried spinning up on it today but hit issues and I'm in a sprint so I don't want to spend time filing them right now :/

I hope Vlad will evolve to be the One True Config that Just Works™ though. I can never fiiiiind oooooone.

**Comment by philipnorton42** [comment]
Sorry for the delay in getting the dev_php_versions branch sorted out. I've managed to get the majority of this work done now (just needs some testing) so I've now merged the branch into dev.
The dev branch now includes support for PHP 5.5 (and 5.6) so I'm going to close this request in favour of issue #130 where we talk about PHP versions and support on each operating system available. If we can draw a line under the dev branch as it stands then we can release version 1.0! :)
